### ISSUE TITLE
facet: false to prevent a mark from being faceted?

### ISSUE THREAD

**Issue by mbostock** [observablehq/plot#291]
Currently we rely on the equality operator to determine whether to facet marks:

https://github.com/observablehq/plot/blob/650529ab865e2e403c1147c77c5371ec6a71e843/src/facet.js#L41

I wonder if in addition to this equality check, we could provide a generic *facet* option, which if set to false would disable faceting for the mark. (I don’t think you’d be able to set it to true if the data is different that the facet, but maybe that would be okay if it has the same cardinality?)

In other words, instead of

```js
Plot.dot(penguins.slice(), {
  x: "culmen_depth_mm",
  y: "culmen_length_mm",
  r: 2,
  fill: "#ddd"
})
```

you’d say

```js
Plot.dot(penguins, {
  facet: false,
  x: "culmen_depth_mm",
  y: "culmen_length_mm",
  r: 2,
  fill: "#ddd"
})
```

(Though, the former would still work…)

**Comment by mbostock** [comment]
Fixed in #450.
